bin/reth/src/commands/debug_cmd/merkle.rs

```diff
@@ -144,7 +145,7 @@ impl<C: ChainSpecParser<ChainSpec = ChainSpec>> Command<C> {
         for block in blocks.into_iter().rev() {
             let block_number = block.number;
             let sealed_block = block
-                .try_seal_with_senders()
+                .try_seal_with_senders::<<N::Primitives as NodePrimitives>::Block>()
```

Comment:
```suggestion
                .try_seal_with_senders::<BlockTy<N>>()
```
add another type alias here for `BlockTy`
https://github.com/paradigmxyz/reth/blob/6695d07c656e7461efeb8cd5e76d54628b1cce20/crates/node/types/src/lib.rs#L236-L243

Reply:
updated in a587d1955